dart/packages/fury/lib/src/serializer/other_core/string_serializer.dart

```diff
@@ -0,0 +1,96 @@
+import 'dart:typed_data';
+import 'package:fury/src/config/fury_config.dart';
+import 'package:fury/src/const/obj_type.dart';
+import 'package:fury/src/deserializer_pack.dart';
+import 'package:fury/src/exception/deserialization_exception.dart';
+import 'package:fury/src/memory/byte_reader.dart';
+import 'package:fury/src/memory/byte_writer.dart';
+import 'package:fury/src/serializer/serializer.dart';
+import 'package:fury/src/serializer/serializer_cache.dart';
+import 'package:fury/src/serializer_pack.dart';
+import 'package:fury/src/util/string_util.dart';
+
```

Comment:
We could leave common serializers under `dart/packages/fury/lib/src/serializer` directly. Only serializers which constructs a group like collection/time should be left into another sub directory